leaflet/index.d.ts

```diff
@@ -145,7 +145,7 @@ declare namespace L {
 
     export class LatLngBounds {
         constructor(southWest: LatLngExpression, northEast: LatLngExpression);
-        constructor(latlngs: LatLngBoundsLiteral);
+        constructor(latlngs?: LatLngBoundsLiteral);
```

Comment:
I checked the source code and this is perfectly valid. I am concerned, however, that allowing it will introduce extra checks when we get around to supporting strict null checks (which people are starting to address in #15039). The problem is that if we allow this to be optional, calls to `getNorthEast` and  `getSouthWest` may return `undefined` and calls to methods like `getCenter` and `getNorth` start throwing. It's a tradeoff that I don't know if people are willing to make.